plugins/modules/vm_snapshot_attach_disk.py

```diff
@@ -242,9 +262,35 @@ def main() -> None:
                 type="int",
                 required=True,
             ),
+
+            # Maybe the user will want to keep the destination VM shut down if, for example he has more tasks on
+            # this VM and wants to make it go quicker by keeping the VM shut down
+            #   - keep or remove this parameter?
+            #       - if keep, what could be a better name for it?
+            #           - dont_reboot_destination_vm
+            #           - keep_destination_vm_shutdown
+            #           - keep_destination_vm_off
+            #           - ignore_destination_vm_reboot --> False: reboot, True: no reboot
+            #           - ...?
+            reboot_destination_vm=dict(
+                type="bool",
+                default=True,
+            ),
```

Comment:
As this big comment explains... should I keep this parameter?

Reply:
I suggest to drop this parameter. Functionally, we get the same result if VM is shut down using `vm_params` module before `vm_snapshot_attach_disk` module is called. I do not see why we would need the extra parameter.